modules/msk/salmon/quant/main.nf

```diff
@@ -0,0 +1,57 @@
+process SALMON_QUANT {
+    tag "$meta.id"
+    label "process_medium"
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://mskcc/salmon:0.14.0' :
+        'docker.io/mskcc/salmon:0.14.0' }"
+
+
+    input:
+    tuple val(meta), path(reads)
+    path  index
+
+    output:
+    tuple val(meta), path("${meta.id}.salmon.quant")    , emit: results
+    tuple val(meta), path("${meta.id}.quant.sf")        , emit: quant
+    path  "versions.yml"                                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args     = task.ext.args   ?: ''
+    def prefix   = task.ext.prefix ?: "${meta.id}"
+    def threads  = task.cpus * 2
+
+    """
+    salmon quant \\
+        --index ${index} \\
+        --seqBias \\
+        --gcBias \\
+        --libType A \\
+        --unmatedReads ${reads} \\
+        --validateMappings \\
+        -o ${prefix}.salmon.quant \\
+        --threads ${threads}
```

Comment:
Do we need to add $args to the end of this?

Reply:
we should!